src/core/viz/expressions/aggregation/clusterAggregation.js

```diff
@@ -7,8 +7,10 @@ import { checkInstance, checkType } from '../utils';
  * Aggregate using the average. This operation disables the access to the property
  * except within other cluster aggregate functions.
  *
- * @param {carto.expressions.prop} property - Column of the table to be aggregated, must be a date or a number
- * @return {carto.expressions.prop} Aggregated column
+ * Note: `clusterAvg` input property can only be a {@link carto.expressions.prop|carto.expressions.prop}, not other expressions.
+ *
+ * @param {Number} property - Column of the table to be aggregated
```

Comment:
we shouldn't name it `property`, since it can be a generic expression (of type Number)

Reply:
This parameter can only be a carto.expression.prop of type Number (this is what the note says). We are on cluster aggregations.

Reply:
ahg, sorry I read `viewport`, my bad